test/test_dummy_sklearn.py

```diff
@@ -0,0 +1,3 @@
+def test_dummy_sklearn():
```

Comment:
You could include imports of the relevant libraries here, thereby testing that they get installed in the testenv during the build (and remove the dummy from the modules name)

Reply:
Done